pkg/apis/networking/v1alpha1/ingress_types.go

```diff
@@ -110,9 +110,6 @@ type IngressSpec struct {
 	// A list of host rules used to configure the Ingress.
 	// +optional
 	Rules []IngressRule `json:"rules,omitempty"`
-
-	// Visibility setting.
-	Visibility IngressVisibility `json:"visibility,omitempty"`
```

Comment:
@tcnghia  Please allow me double check. We can remove it now ? Or we should add "obsolete" comment and remove it after 9 months?  

I am worried as some KIngress still refer to this field, so they need to update as soon as this change was pulled.

e.g. contour
https://github.com/knative-sandbox/net-contour/blob/31f50d1c2c86d19245d42b34a9532a833daf990d/pkg/reconciler/contour/resources/kingress.go#L46

Reply:
Sincerely, we can keep my change https://github.com/knative/serving/pull/8984 which only involve serving. I suggest don't remove  spec.Visibility temporary.